docs/contributing.md

```diff
@@ -100,6 +100,15 @@ You can also run a single test script like this:
 $ nox -s test -- tests/test_multipart.py
 ```
 
+Lastly, to ensure you're on track to pass the CI build, run:
+
+```shell
+# Note: you may need to run `$ chmod +x scripts/test` once beforehand.
```

Comment:
We shouldn't need to do that.
I think git handles file permissions, so make sure they're set correctly in this PR.

Reply:
Oh, cool. I’ll try to check them in then. :)

Reply:
I didn't know Git checked in file permissions — turns out I had already got them into this PR! I removed this comment.